README.md

```diff
@@ -185,6 +185,9 @@ Modules in this repository can be used independently:
 ```hcl
 module "kubernetes" {
   source = "github.com/hobby-kube/provisioning/service/kubernetes"
+
+  # Or to pin a certain commit
+  # source = "git::https://github.com/hobby-kube/provisioning.git//service/kubernetes?ref=800d5d5031245cf31a803a147eaa40a0de0573f1"
```

Comment:
double slash after .git is probably wrong here

Reply:
That is specifically needed to use the subdirectory of a repo, see [Terraform docs: Modules in Package Sub-directories](https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories).

Reply:
my fault. thanks for clarifying.